src/protocol/bolt/src/main/java/org/apache/jmeter/protocol/bolt/sampler/AbstractBoltTestElement.java

```diff
@@ -48,4 +84,27 @@ public boolean isRecordQueryResults() {
     public void setRecordQueryResults(boolean recordQueryResults) {
         this.recordQueryResults = recordQueryResults;
     }
+
+    //returns a SessionConfig object that can be passed to the driver session
+    public SessionConfig getSessionConfig() {
+        SessionConfig.Builder sessionConfigBuilder = SessionConfig.builder()
+                .withDefaultAccessMode(Enum.valueOf(AccessMode.class, getAccessMode()));
+
+        if (database != null && !"".equals(database)) {
```

Comment:
You could probably use `StringUtils#isNotBlank(String)` to make intention more clear.